vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
+- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+
+### Minimum Hardware Requirements
+
+The minimum hardware requirements depends on how you intend to use Entando 6. The more functionality you need such as a content management system (CMS), the more you will need.
```

Comment:
depend (not plural)
Eliminate second person.
Comma should follow "need."

Reply:
Right on two counts.
Not sure about the second person.

Reply:
me neither! i think the main issue is that it's very inconsistently used, sometimes even within the same section. i guess it depends on how formal we want to be... for example, "...your Entando installation" is very conversational, and frowned upon by some, but "...the user's Entando installation" is almost too uptight.